src/App.tsx

```diff
@@ -15,71 +16,91 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGood: string | null;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  selectedGoods = (good: string) => {
```

Comment:
so, it's function return a selected good? Array?
![image](https://user-images.githubusercontent.com/91411692/232804083-c85529d2-d897-482d-8601-411ff0dd1fb0.png)
```suggestion
  handleSelect = (good: string) => {
```

Reply:
Fix it